packages/eslint-config-typescript/react.js

```diff
@@ -0,0 +1,100 @@
+/**
```

Comment:
This file needs to be added to the [doc](https://github.com/strvcom/code-quality-tools/blob/master/packages/eslint-config-react/readme.md#L11) as well! 🙏